Dockerfile.alpine

```
@@ -25,7 +28,13 @@ COPY src/D2L.Bmx/D2L.Bmx.csproj src/D2L.Bmx/D2L.Bmx.csproj
 RUN dotnet restore src/D2L.Bmx -r linux-musl-x64
 
 COPY . .
-RUN dotnet publish src/D2L.Bmx --no-restore -r linux-musl-x64 -o app
+RUN dotnet publish src/D2L.Bmx \
+    -p:Version=$version \
+    -p:InformationVersion=$information_version \
```

variable references should be quoted in shell scripts as a best practice
```suggestion
    -p:Version="$version" \
    -p:InformationVersion="$information_version" \
```